frontend/src/modules/staff/components/orders/OrderCard.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import {
+  Card,
+  CardContent,
+  CardActions,
+  IconButton,
+  Typography,
+  Box,
+  Divider,
+  Chip,
+  FormControl,
+  Select,
+  MenuItem,
+  Collapse,
+  List,
+  ListItem,
+  ListItemText,
+  SelectChangeEvent,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import ExpandLessIcon from "@mui/icons-material/ExpandLess";
+import CloseIcon from "@mui/icons-material/Close";
+import { Order, OrderStatus, statusColors } from "./types";
+
+type Props = {
+  order: Order;
+  expanded: boolean;
+  handleExpandClick: (id: number) => void;
+  handleStatusChange: (
+    id: number,
+    event: SelectChangeEvent<OrderStatus>
+  ) => void;
+  handleRemoveOrder: (id: number) => void;
+};
+
+const OrderCard: React.FC<Props> = ({
+  order,
+  expanded,
+  handleExpandClick,
+  handleStatusChange,
+  handleRemoveOrder,
+}) => {
+  return (
+    <Card variant="outlined" sx={{ m: 1, boxShadow: 3, overflow: "visible" }}>
```

Comment:
What do you think of using this box shadow? We've been using it in other parts of the app.

```suggestion
    <Card variant="outlined" sx={{ m: 1, boxShadow: "1px 2px 7px 1px " + blueGrey[100] + "99", overflow: "visible" }}>
```

Reply:
looks super nice